models/src_zendesk.yml

```diff
@@ -115,6 +117,7 @@ sources:
 
       - name: organization_tag
         description: The tags associated with an organization. An organization may have multiple tags.
+        freshness: null
```

Comment:
Curious what the reason for setting the organization_tag freshness to null is? I agree that users probably don't need this to be in line with the freshness standards. I do wonder if this was a request from a customer. I checked our data and noticed our freshness for organization tags is in line with our freshness period defined in this yml.

Reply:
yeah that was in response to this [issue](https://github.com/fivetran/dbt_zendesk_source/issues/19)

Reply:
ah i was looking at an old zendesk schema that we probably haven't synced in a while, so i didn't see that our `organization_tag` was actually maintaining freshness

Reply:
however in the customer's case they do have this table and it's not up to freshness standards, which makes me lean toward setting it to null? not sure if this is the case for other folks